data/operators/amenity/library.json

```diff
@@ -873,6 +873,62 @@
         "operator:wikidata": "Q6458744"
       }
     },
+    {
+      "displayName": "London Borough of Hounslow",
+      "id": "londonboroughofhounslow-d8f64f",
```

Comment:
Could you remove the `id` for the entries as this is done when a build is created

Reply:
I ran `npm run build` myself because the wiki says I could do so if possible, but I will remove the ids if it is simpler.